### PR TITLE
A lot of minor fixes

### PR DESCRIPTION
**Pull request by michaelowens**
Going through the code I noticed a lot of indentation or inconsistencies. I tried getting rid of some unnecessary return statements, indents, duplicated code, etc.

Trying to keep the changes low impact, so probably things over the next few days I'm going to look for some more minor things I could change and then this should be ready to merge.

### CODE REVIEW

**Comment by Maxorq** [comment]
that's nice, I wonder how it's going to fit with @vendethiel's continued progress in breaking the entire thing though

**Comment by vendethiel** [comment]
Might not be that bad

**Comment by Maxorq** [comment]
hue

**Comment by michaelowens** [comment]
Are you doing some kind of refactor/overhaul of a lot of code, @vendethiel ? I'm trying to keep all changes in this low impact, thus more refactoring some if's and small changes here and there. Don't want this to be a PR that has such huge changes and not be merged because of that.

**Comment by vendethiel** [comment]
I'll push that somewhere so I can show you, I guess.

**Comment by michaelowens** [comment]
That would be nice, I could run a patch of your changes against mine and see if it causes any issues.

**Comment by vendethiel** [comment]
#19

**Comment by Maxorq** [comment]
it's funny because I'm not even involved

**Comment by vendethiel** [comment]
feel free to refactor other stuff as well. 3-way merges are so fun...

**Comment by michaelowens** [comment]
There's quite some overal in changes with such a big refactor going on. Might want to merge/take those changes to @vendethiel 's PR rather than changing them here and trying to merge them back into ven's ... Would save him a lot of headaches merging, I guess

File: core/Tools.iced [hunk 1]
```diff
@@ -14,19 +13,19 @@ class exports.Tools
 	colorRarity: (rarity) ->
 		switch rarity
 			when 'common'
-				return '<span style="color: white;">Common</span>';
```
Comment (vendethiel) [inline core/Tools.iced]: This is a fight I didn't make

File: models/Badge.iced [hunk 2]
```diff
@@ -27,8 +26,7 @@ class exports.Badge extends Mikuia.Model
 		await @_smembers 'members', defer err, data
 		callback err, data
 
-	getName: (callback) ->
-		return @name
+	getName: (callback) -> @name
```
Comment (vendethiel) [inline models/Badge.iced]: why is that even taking a callback again

File: models/Leaderboard.iced [hunk 3]
```diff
@@ -1,9 +1,8 @@
 class exports.Leaderboard extends Mikuia.Model
-	constructor: (name) ->
+	constructor: (@name) ->
 		@model = 'leaderboard'
-		@name = name
 
-		Mikuia.Element.register 'leaderboards', name
+		Mikuia.Element.register 'leaderboards', @name
```
Comment (vendethiel) [inline models/Leaderboard.iced]: I'm moving Elements around

Comment (vendethiel) [inline models/Leaderboard.iced]: should not be needed

File: plugins/base/plugin.iced [hunk 4]
```diff
@@ -160,9 +160,9 @@ Mikuia.Events.on 'twitch.message', (from, to, message) =>
...
 		return true
-	else
-		moderators = Mikuia.Chat.mods channel
-		if moderators? && username in moderators
-			return true
-		else
-			return false
\ No newline at end of file
+
+	moderators = Mikuia.Chat.mods channel
```
Comment (vendethiel) [inline plugins/base/plugin.iced]: probably didn't make this change

File: models/Badge.iced (hunk 2 above)
Comment (michaelowens) [inline models/Badge.iced]: I didn't even pay attention to that, lmao

**Comment by michaelowens** [comment]
@vendethiel maybe it's worth running https://github.com/Maxorq/Mikuia/pull/18.patch against your branch, ignoring/choosing your changes at any merges and accepting the changes near the bottom. Then this could be closed.

**Comment by vendethiel** [comment]
Patch does not apply :P.

**Comment by vendethiel** [comment]
I think I got it :+1: (I commited with `--author="Michael Owens <michaelowensnl@gmail.com>"` obviously)

**Comment by michaelowens** [comment]
Awesome! I'll check it out soon, I'm in the middle of a move so I'm kind of busy this week.

File: plugins/base/plugin.iced (hunk 4 above)
Comment (vendethiel) [inline plugins/base/plugin.iced]: should actually use `Channel.isModOf` tho?

Comment (Maxorq) [inline plugins/base/plugin.iced]: THIS CODE IS SO CONSISTENT LOL

**Pull request closed without merging**